src/Adapter/Presenter/AbstractLazyArray.php

```diff
@@ -68,6 +68,8 @@
  */
 abstract class AbstractLazyArray implements Iterator, ArrayAccess, Countable, JsonSerializable
 {
+    private const INDEX_NAME_PATTERN = '/@indexName\s+"([^"]+)"/';
```

Comment:
this is temporary, a next PR will replace all usages of phpdoc with new php attribute feature